types/index.d.ts

```diff
@@ -44,6 +44,8 @@ declare namespace RamdaAdjunct {
     interface Dictionary<T> { [key: string]: T; }
 
     type DictPred<T> = (value: T, key: string) => boolean;
+    /* tslint:disable-next-line:no-null-undefined-union null or undefined is the accurate type */
```

Comment:
We should think about removing this lint rule - `null | undefined` is a good type for cases like this where you're doing type-narrowing. A similar issue arose in https://github.com/char0n/ramda-adjunct/pull/1672

Reply:
Let's do it. I have no strong opinions about typescript